examples/web-rest-openapi-simple.ps1

```diff
@@ -40,14 +40,14 @@ Start-PodeServer {
 
     Add-PodeRoute -Method Post -Path '/api/users' -ScriptBlock {
         param($e)
-        Write-PodeJsonResponse -Value @{ Name = 'Rick'; UserId = $e.Data.userId }
+        Write-PodeJsonResponse -Value @{ Name = $e.Data.Name; UserId = $e.Data.userId }
```

Comment:
Agree with the `$e.Data.Name`.

For the `userId`, this was likely a copy-paste error 🙈 To match the `Name` format this could be best off being renamed to `UserId` - leaving the below OA definition as `UserId` 🤔